packages/libs/sdk-mixins/src/mixins/descopeUiMixin/constants.ts

```diff
@@ -2,14 +2,20 @@ import { IS_LOCAL_STORAGE } from '../../constants';
 
 export const DESCOPE_UI_SCRIPT_ID = 'load-descope-ui';
 export const DESCOPE_UI_FALLBACK_SCRIPT_ID = 'load-descope-ui-fallback-script';
+export const DESCOPE_UI_FALLBACK_2_SCRIPT_ID =
+  'load-descope-ui-fallback-script';
```

Comment:
We cannot have the same ID twice
```suggestion
  'load-descope-ui-fallback-script-2';
```